docs/microservices.json

```diff
@@ -2896,6 +2900,59 @@
       ],
       "apis": [],
       "version": null
+    },
+    {
+      "id": "am-judicial-booking-service",
+      "name": "AM Judicial Booking Service",
+      "group": "AM",
+      "type": "api",
+      "description": "API will help to create bookings for fee paid judges and would provided APIs to Fetch and create booking.",
+      "repository": "https://github.com/hmcts/am-judicial-booking-service",
+      "spec": "https://hmcts.github.io/cnp-api-docs/specs/am-judicial-booking-service.json",
+      "dependencies": [],
+      "apis": [],
+      "version": null
+    },
+    {
+      "id": "am-org-role-mapping-service",
+      "name": "AM Organisational Role Mapping Service",
+      "group": "AM",
+      "type": "api",
+      "description": "API to provision various organisation roles for staff & judicial users based on the service specific mapping rules.",
+      "repository": "https://github.com/hmcts/am-org-role-mapping-service",
+      "spec": "https://hmcts.github.io/cnp-api-docs/specs/am-org-role-mapping-service.json",
+      "dependencies": [
```

Comment:
We can also add dependencies to:
* "rd-judicial-api"
* "rd-caseworker-ref-api"

and as most other common services have added IDAM and S2S to their dependecnies we could on all 3 microservices **as the first two dependencies**: 
* "idam-api",
* "rpe-service-auth-provider"

Reply:
Dependencies added and reordered